plugins/utils/dashboard.nix

```diff
@@ -14,127 +14,370 @@ in {
 
       package = helpers.mkPackageOption "dashboard" pkgs.vimPlugins.dashboard-nvim;
 
-      header = mkOption {
-        description = "Header text";
-        type = types.nullOr (types.listOf types.str);
-        default = null;
+      theme =
+        helpers.defaultNullOpts.mkNullable
+        (with types; either (enum ["hyper"]) (enum ["doom"]))
+        "hyper"
+        "The dashboard's theme.";
+
+      disableMove =
+        helpers.defaultNullOpts.mkBool
+        true
+        "Whether to disable the move keys";
+
+      shortcutType =
+        helpers.defaultNullOpts.mkNullable
+        (with types; either (enum ["letter"]) (enum ["number"]))
+        "letter"
+        "The shortcut type.";
+
+      changeVCSRoot =
+        helpers.defaultNullOpts.mkBool
+        false
+        "Change the root of VCS for the `hyper` theme's MRU module.";
+
+      header =
+        helpers.defaultNullOpts.mkListOf
```

Comment:
If default is null we avoid using the defaultNullOpts heplers

Instead something like `helpers.mkNullOrOption (with nixvimTypes; listOf (maybeRaw str)))`